go/vt/vttablet/tabletserver/vstreamer/rowstreamer.go

```diff
@@ -260,26 +260,44 @@ func (rs *rowStreamer) buildSelect(st *binlogdatapb.MinimalTable) (string, error
 		}
 		prefix = ", "
 	}
+
+	addPushdownExpressions := func() {
+		for i, expr := range rs.plan.whereExprsToPushDown {
+			if i != 0 {
+				// Only AND expressions are supported.
+				buf.Myprintf(" and ")
+			}
+			buf.Myprintf("%s", sqlparser.String(expr))
```

Comment:
I'd feel safer with added parenthesis, though admittedly I can't see a clear scenario where it would be required. Take it or leave it.
```suggestion
			buf.Myprintf("(%s)", sqlparser.String(expr))
```

Reply:
I don't think it's needed but it certainly doesn't hurt so I can add it.